syftlib/src/main/java/org/openmined/syft/SignallingInterface.kt

```diff
@@ -0,0 +1,20 @@
+package org.openmined.syft
+
+interface SignallingInterface {
+
+    /**
+     * Called by the UI activity as a response to message received from PyGrid
```

Comment:
I think it's `Called IN the UI...`  or `Implemented by the UI...`

Reply:
Ohh yeah 😅